src/components/dashboard/Dashboard.js

```diff
@@ -660,53 +803,91 @@ const Dashboard = props => {
+                      <Avatar
+                        source={avatar}
+                        style={styles.avatar}
+                        imageStyle={styles.avatar}
+                        unknownStyle={styles.avatar}
+                        plain
+                      />
+                    </TouchableOpacity>
+                  </Animated.View>
+                  <Animated.View style={[styles.headerFullName]}>
+                    <Section.Text color="gray100Percent" fontFamily={theme.fonts.default} fontSize={16}>
+                      {fullName || ' '}
+                    </Section.Text>
+                  </Animated.View>
+                </Animated.View>
+              </Section>
+              <Animated.View style={balanceAnimStyles}>
+                <Text
+                  color="gray100Percent"
+                  fontFamily={theme.fonts.default}
+                  fontSize={12}
+                  style={{ marginTop: 8, marginBottom: 8 }}
+                >
+                  {` MY TOTAL BALANCE `}
+                </Text>
+                <View onLayout={onBalanceLayout} style={styles.balanceUsdRow}>
+                  <BigGoodDollar
+                    testID="amount_value"
+                    number={balance}
+                    formatter={balanceFormatter}
+                    bigNumberStyles={[styles.bigNumberStyles, calculateFontSize]}
+                    bigNumberUnitStyles={styles.bigNumberUnitStyles}
+                    bigNumberProps={{
+                      numberOfLines: 1,
+                    }}
+                    style={styles.bigGoodDollar}
+                  />
+                </View>
+                <Text style={styles.gdPrice}>
+                  ≈ {calculateUSDWorthOfBalance} USD <GoodDollarPriceInfo />
+                </Text>
+              </Animated.View>
             </Animated.View>
-            <Animated.View style={[styles.bigNumberWrapper, balanceAnimStyles]}>
-              <View onLayout={onBalanceLayout}>
-                <BigGoodDollar
-                  testID="amount_value"
-                  number={balance}
-                  formatter={balanceFormatter}
-                  bigNumberStyles={[styles.bigNumberStyles, calculateFontSize]}
-                  bigNumberUnitStyles={styles.bigNumberUnitStyles}
-                  bigNumberProps={{
-                    numberOfLines: 1,
-                  }}
-                  style={styles.bigGoodDollar}
-                />
+
+            <Animated.View style={multiBalanceAnimStyles}>
+              <View style={[styles.multiBalance, { marginTop: headerLarge ? 0 : 10, flexDirection: 'row' }]}>
+                <Section style={styles.multiBalanceItem}>
+                  <Text style={{ fontSize: 16 }}>{celoBalance}</Text>
+                  <Text style={{ fontSize: 12 }}>Celo G$</Text>
+                </Section>
+                <Section.Text style={[styles.gdPrice, gdPriceAnimStyles, { width: '40%', fontSize: 20 }]}>
```

Comment:
try to get rid of inline styles if they're not containing calculations

Reply:
done